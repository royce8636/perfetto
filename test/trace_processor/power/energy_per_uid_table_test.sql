SELECT consumer_id
FROM energy_per_uid_counter_track
JOIN uid_counter_track ON energy_per_uid_counter_track.id = uid_counter_track.id