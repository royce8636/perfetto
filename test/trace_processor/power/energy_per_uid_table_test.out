"consumer_id"
3
